openedx_caliper_tracking/pavelib/tests.py

```diff
@@ -0,0 +1,53 @@
+import os
+import json
+import requests
+from datetime import datetime
+
+from paver.easy import task
+
+from pavelib.utils import loggers
+from pavelib.settings import CALIPER_TESTS_API_ADDRESS, CALIPER_TESTS_API_TOKEN
+
+TEST_DIR_PATH = 'tests/'
+TESTS_LOG_FILE = 'caliper_tests.log'
+
+
+logger = loggers.get_test_logger('caliper_tests', TESTS_LOG_FILE)
+
+
+@task
+def test_caliper_compliance():
+    test_files = [
+        file for file in os.listdir(
+            '{}expected/'.format(TEST_DIR_PATH)
+        ) if file.endswith(".json")
+    ]
+    for file in test_files:
+        test_json = json.load(
+            open('{}expected/{}'.format(TEST_DIR_PATH, file), 'r'))
+        if not '@context' in test_json:
```

Comment:
Why is this check necessary?

Reply:
Because there are some json files in `expected` directory which are not transformed into caliper format due to lack of required data in input events. Those events are in legacy format and thus should not be tested for caliper compliance.